### PR TITLE
solution for exo67

### PR DESCRIPTION
**Pull request by tianyuchen**

### CODE REVIEW

File: 67-Add_Binary.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    def addBinary(self, a: str, b: str) -> str:
+        carry = 0
+        res = ""
+
+        a = list(a)
+        b = list(b)
+
+        while a or b or carry:
+            if a:
+                carry += int(a.pop())
```
Comment (gameboy1024) [inline 67-Add_Binary.py]: int(some_str) is slow, you can just do:
if a and a.pop() == '1':
  carry += 1

File: 67-Add_Binary.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            if b:
+                carry += int(b.pop())
+            res += str(carry % 2)
```
Comment (gameboy1024) [inline 67-Add_Binary.py]: String concatenation is slow. You should store the result in an array and call "".join(res) instead.

File: 67-Add_Binary.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+            carry = carry // 2
```
Comment (gameboy1024) [inline 67-Add_Binary.py]: You can use bitwise shift to be quicker:
carry >>= 1

**Review by gameboy1024 — CHANGES_REQUESTED** [review]

File: 67-Add_Binary.py (hunk 1 above)
Comment (tianyuchen) [reply 67-Add_Binary.py]: already changed

**Review by gameboy1024 — APPROVED** [review]

File: 67-Add_Binary.py (hunk 2 above)
Comment (tianyuchen) [reply 67-Add_Binary.py]: changed :)

File: 67-Add_Binary.py (hunk 3 above)
Comment (tianyuchen) [reply 67-Add_Binary.py]: thx

**Pull request merged**